CMakeLists.txt

```diff
@@ -265,11 +265,11 @@ set(SWIFT_ANDROID_DEPLOY_DEVICE_PATH "" CACHE STRING
   "Path on an Android device where build products will be pushed. These are used when running the test suite against the device")
 
 #
-# User-configurable ICU specific options for Android, FreeBSD, Linux and Haiku.
+# User-configurable ICU specific options for Android, FreeBSD, Linux, Haiku, and WebAssembly.
```

Comment:
s/WebAssembly/WASI/ please.  I dont think it makes sense to support WebAssembly as much as it does to support WASI.  WASI is much closer as a target to the rest of the currently supported targets.  While I am interested in supporting WebAssembly, it is closer to a baremetal target and IMO it is better to support a better supported architecture (like ARM64) on bare metal before doing WebAssembly.